### DISCUSSION TITLE
How to Kill designated job ?

### DISCUSSION THREAD

**Discussion by zfung03** [swar/Swar-Chia-Plot-Manager#1076] [General]
How to Kill designated job ?

As I temporary driver disk full. how to kill designated job and I can del temporary file. thanks

**Comment by FlyingToto** [comment]
on linux you kill the job with the process ID listed in the pid column, but before you kill it, note the plotid string so you can use it to delete the files with the matching pattern in the name.
kill 12345
rm /tmpdrive/plots/*acbcd*plot*tmp
